src/runtime/virtcontainers/qemu.go

```diff
@@ -1007,6 +1014,29 @@ func (q *qemu) cleanupVM() error {
 		}
 	}
 
+	if rootless.IsRootless() {
+		rootlessDir := os.Getenv("XDG_RUNTIME_DIR")
+		if rootlessDir != "" {
```

Comment:
I get that you set XDG_RUNTIME_DIR and here you're basically ensuring that your userTmpDir was created and then removing it.  However, what's the case that `os.Getenv("XDG_RUNTIME_DIR")` would return `""`?

I think this check may be a no-op and we'd endup either deleting the userTmpDir or whater is under `/run/user/uid`.  Am I missing something? (It's very much possible, by the way ;-))

Reply:
You're right. The empty string check is not necessary. Removed.